lib/game_client/ogs/ogs_websocket_manager.dart

```diff
@@ -135,9 +162,36 @@ class OGSWebSocketManager {
       final data = jsonDecode(messageStr);
       
       if (data is List && data.isNotEmpty) {
-        final command = data[0] as String;
+        final commandOrId = data[0];
         final payload = data.length > 1 ? data[1] : null;
-        final requestId = data.length > 2 ? data[2] : null;
```

Comment:
This was wrong before. Request ID is the first element if it exists.

The third element is an error if it exists.